.github/workflows/submodules-sync.yml

```diff
@@ -6,10 +6,17 @@ name: Submodules Sync
 
 on:
   workflow_call:
+    inputs:
+      scripts_directory:
```

Comment:
For a user to formulate the `scripts_directory` input they need to know two things:
1. The path to the `dev-tools` submodule. That's something they choose, so is their responsibility.
2. The path to the `scripts` folder inside `dev-tools`. This part is not something they've chosen, it's something we've chosen.  So it's strange to make it their responsibility to figure out. In fact, it's problematic if we ever decide to change the location of the scripts inside `dev-tools` - all of the users would have to update their code.

How about we instead only ask the user for an input like `devtools_directory` (e.g. `"submodules/dev-tools"`) and we ourselves deduce that `SCRIPTS_DIRECTORY` will be e.g. `${{ inputs.devtools_directory }}/.github/workflow/scripts`.

Reply:
Fixed.

Reply:
Added changes to both PRs https://github.com/reboot-dev/respect/pull/354 and https://github.com/3rdparty/eventuals/pull/359 accordingly.